terraform/environments/digital-prison-reporting/modules/domains/replay-pipeline/pipeline.tf

```diff
@@ -10,8 +10,31 @@ module "replay_pipeline" {
   definition = jsonencode(
     {
       "Comment" : "Replay Pipeline Step Function",
-      "StartAt" : "Stop DMS Replication Task",
+      "StartAt" : "Deactivate Archive Trigger",
       "States" : {
+        "Deactivate Archive Trigger" : {
```

Comment:
At start of the replay pipeline, the trigger for the archive job is deactivated.